backend/app/tests/utils/collection.py

```
@@ -17,25 +18,81 @@ class constants:
 
 
-def get_collection(db: Session, client=None, project_id: int = None) -> Collection:
-    project = get_project(db)
-    if client is None:
-        client = OpenAI(api_key="test_api_key")
+def get_collection(
+    db: Session,
+    project,
+    *,
+    assistant_id: Optional[str] = None,
+    model: str = "gpt-4o",
```

Ruff (0.14.0)

Unused function argument: `model`

(ARG001)